application/backup.sh

```diff
@@ -151,4 +107,8 @@ if [ "${B2_BUCKET}" != "" ]; then
     fi
 fi
 
+echo "postgresql-backup-restore: backup: Completed"
+
 log "${MYNAME}: backup: Completed"
+
+exit $STATUS
```

Comment:
Line termination is missing.
```suggestion
exit $STATUS
```

Reply:
noted, made sure to use proper termination through.